src/workerd/api/worker-rpc.c++

```diff
@@ -54,7 +54,7 @@ kj::Promise<capnp::Response<rpc::JsRpcTarget::CallResults>> WorkerRpc::sendWorke
     auto ser = serializeV8(js, js.arr(argv.asPtr()));
     // TODO(soon): We will drop this requirement once we support streaming.
     JSG_ASSERT(ser.size() < MAX_JS_RPC_MESSAGE_SIZE, Error,
-        "Serialized RPC request is too large: ", ser.size(), " <= ", MAX_JS_RPC_MESSAGE_SIZE);
+        "Serialized RPC request is too large: ", MAX_JS_RPC_MESSAGE_SIZE, " <= ", ser.size());
```

Comment:
This message formatting is probably still unclear to users, since it doesn't really say which number is the limit vs. the actual. Maybe needs to be something like:

> Serialized RPC requests are limited to 1MB, but the size of this request was: 12345678